api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -18,53 +18,38 @@ package v1alpha1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	vrv1alpha1 "github.com/kube-storage/volume-replication-operator/api/v1alpha1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
-// VolumeTakeoverControlType -- takeover control when desiredCluster is different from affinedCluster
-type VolumeTakeoverControlType string
-
-// VolumeTakeoverControlType definitions
-const (
-	// Force promote the volume in a WAN DR setting
-	ForcePromote VolumeTakeoverControlType = "ForcePromote"
-)
-
-// VolumeReplicationGroupSpec defines the desired state of VolumeReplicationGroup
+// VolumeReplicationGroup (VRG) spec declares the desired replication class
+// and replication state of all the PVCs owned by a given application, which
+// are identified via the given PVC label selector.  For every PVC created
+// by the application, the VRG operator does the following:
```

Comment:
(nit) reword operator to reconciler

Reply:
I agree that the term reconciler is accurate but from the perspective of someone reading the comments via pkg.go generated webpage, will the term operator be easier to follow?

Reply:
I removed the operator word altogether to result in: "For each such PVC, the VRG will do the following:"